public/options/options.html

```diff
@@ -13,608 +14,478 @@
-						<span class="slider round"></span>
-					</label>
-					<div class="switch-label">
-						__MSG_supportOtherSites__
-					</div>
-				</label>
-	
-				<br/>
-				<br/>
-				<br/>
-	
-				<div class="small-description">(__MSG_supportedSites__ Invidious, CloudTube)</div>
-				<br/>
-				<span class="small-description">__MSG_supportOtherSitesDescription__ </span>
+		</div>
 
-				<br/>
-				<br/>
-				<br/>
-			</div>
-	
-			<div option-type="private-text-change" sync-option="invidiousInstances" no-safari="true">
-				<div class="option-button trigger-button">
-					__MSG_addInvidiousInstance__
-				</div>
-	
-				<br/>
-	
-				<div class="small-description">__MSG_addInvidiousInstanceDescription__</div>
-	
-				<div class="option-hidden-section hidden">
-					<br/>
-	
-					<input class="option-text-box" type="text">
-	
-					<br/>
-					<br/>
-	
-					<div class="option-button text-change-set inline">
-						__MSG_add__
-					</div>
+		<div id="options" class="hidden">
 
-					<div class="option-button invidious-instance-reset inline">
-						__MSG_resetInvidiousInstance__
-					</div>
-					
-					<br/>
-					<br/>
+			<div id="segments" class="option-group hidden">
+
+				<div id="category-type" data-type="react-CategoryChooserComponent">
 
-					<span class="small-description">__MSG_currentInstances__</span>
-					<span class="small-description" option-type="display" sync-option="invidiousInstances"></span>
 				</div>
 
-				<br/>
-				<br/>
 			</div>
 
-			<div option-type="keybind-change" sync-option="skipKeybind">
-				<div class="option-button trigger-button">
-					__MSG_setSkipShortcut__
+			<div id="behavior" class="option-group hidden">
+
+				<div data-type="toggle" data-sync="autoSkipOnMusicVideos">
+					<div class="switch-container">
+						<label class="switch">
+							<input id="autoSkipOnMusicVideos" type="checkbox" checked>
+							<span class="slider round"></span>
+						</label>
+						<label class="switch-label" for="autoSkipOnMusicVideos">
+							__MSG_autoSkipOnMusicVideos__
+						</label>
+					</div>
 				</div>
 	
-				<div class="option-hidden-section hidden">
-					<br/>
-	
-					<div class="medium-description keybind-status">
-						__MSG_keybindDescription__
+				<div data-type="toggle" data-sync="muteSegments">
+					<div class="switch-container">
+						<label class="switch">
+							<input id="muteSegments" type="checkbox" checked>
+							<span class="slider round"></span>
+						</label>
+						<label class="switch-label" for="muteSegments">
+							__MSG_muteSegments__
+						</label>
+					</div>
+				</div>
+				
+				<div option-type="toggle" data-sync="fullVideoSegments">
+					<div class="switch-container">
+						<label class="switch">
+							<input id="fullVideoSegments" type="checkbox" checked>
+							<span class="slider round"></span>
+						</label>
+						<label class="switch-label" for="fullVideoSegments">
+							__MSG_fullVideoSegments__
+						</label>
 					</div>
```

Comment:
I put it under mute, but should this be in Interface instead? It doesn't affect anything but whether it shows the icon, right?

Reply:
I feel that this is like enabling or disabling a skip option since this is another way to label things.

Reply:
The three relevant tabs I created are Categories, Behavior and Interface.
I only put the actual categories into Categories. (I'm assuming that's what you mean by skip option?)
I put "Auto skip all segments when there is a non-music segment" and "Allow segments that mute audio instead of skip" into Behavior. This option doesn't technically change skipping behavior, but I suppose it's still a functionality, so it makes sense for it to be there.
Or would you want these 3 options to be on the same tab as the categories?

Reply:
Just took a look, could those three options all be with the categories at the bottom?

Reply:
okay